dim/requirements.txt

```diff
@@ -4,6 +4,7 @@ ldap3~=2.9
 Werkzeug>=0.15.3
 Flask~=1.1
 Flask-SQLAlchemy~=2.5
+click~=7.1
```

Comment:
Where is this dependency used? If this is for development only, can we mark it as such?

Reply:
`Flask` requires `click`, but seems to have strict dependencies already, seems that pip manages to resolve dependencies without that.
Removed it.